src/main/java/net/glowstone/block/data/state/StateUtil.java

```diff
@@ -0,0 +1,21 @@
+package net.glowstone.block.data.state;
+
+import org.bukkit.block.BlockFace;
+
+public interface StateUtil {
+
+    BlockFace[] SIXTEEN_BLOCK_FACES = {BlockFace.SOUTH, BlockFace.SOUTH_SOUTH_WEST, BlockFace.SOUTH_WEST, BlockFace.WEST_SOUTH_WEST, BlockFace.WEST, BlockFace.WEST_NORTH_WEST, BlockFace.NORTH_WEST, BlockFace.NORTH_NORTH_WEST, BlockFace.NORTH, BlockFace.NORTH_NORTH_EAST, BlockFace.NORTH_EAST, BlockFace.EAST_NORTH_EAST, BlockFace.EAST, BlockFace.EAST_SOUTH_EAST, BlockFace.EAST, BlockFace.EAST_SOUTH_EAST, BlockFace.SOUTH_EAST, BlockFace.SOUTH_SOUTH_EAST};
```

Comment:
There are 18 block faces here, not 16. Due to this,w ill your logic actually work as intended?

Reply:
Looks like EAST and EAST_SOUTH_EAST are duplicated.

Reply:
Thanks for spotting that. 

From the cleanup ive been doing locally, Ive been focusing on the ids and default values of the blocks